### PR TITLE
Redcore 462 phpunit

### PR DESCRIPTION
**Pull request by julienV**

### CODE REVIEW

File: build/gulpfile.js [hunk 1]
```diff
@@ -4,6 +4,12 @@ var extension  	= require('./package.json');
 var config      = require('./gulp-config.json');
 
 var argv       	= require('yargs').argv;
+
+if (argv.wwwDir)
```
Comment (Kixo) [inline build/gulpfile.js]: why not use here:
var config = require('./config.js');

File: tests/unit/suite/library/currencyTest.php [hunk 2]
```diff
@@ -10,8 +10,11 @@
 // Register library prefix
 require_once JPATH_LIBRARIES . '/redcore/bootstrap.php';
 
+// Register the classes for autoload.
+JLoader::registerPrefix('R', JPATH_REDCORE);
+
 // Bootstraps redCORE
-RBootstrap::bootstrap(false);
+//RBootstrap::bootstrap(false);
```
Comment (Kixo) [inline tests/unit/suite/library/currencyTest.php]: remove

File: tests/unit/suite/site/soapTest.php [hunk 3]
```diff
@@ -1,85 +0,0 @@
-<?php
```
Comment (Kixo) [inline tests/unit/suite/site/soapTest.php]: this is my soap testing ground :)
I know it doesnt belong here so I will move it to my tools folder in local...

File: build/gulpfile.js (hunk 1 above)
Comment (julienV) [inline build/gulpfile.js]: sure, could be used there too, it's just i did this for a component using redcore, not for redcore standalone release script, so i didn't think of it

File: tests/unit/suite/library/currencyTest.php (hunk 2 above)
Comment (julienV) [inline tests/unit/suite/library/currencyTest.php]: indeed

File: tests/unit/suite/site/soapTest.php (hunk 3 above)
Comment (julienV) [inline tests/unit/suite/site/soapTest.php]: yes, it was massively failing ;)

**Comment by Kixo** [comment]
looks good. Nice job @julienV

**Pull request merged**